testing/src/main/scala/org/http4s/testing/ErrorReportingUtils.scala

```diff
@@ -12,7 +12,7 @@ import java.io.{ByteArrayOutputStream, PrintStream}
 
 import scala.util.control.NonFatal
 
-@deprecated("Not related to HTTP, and will be removed from public interface.", "0.21.0-RC2")
+//@deprecated("Not related to HTTP, and will be removed from public interface.", "0.21.0-RC2")
```

Comment:
There is a copy in `ErrorReporting` that's in the test sources, but available internally.  If you use that, we can leave this one deprecated.  This one is in main, and it's not an API that we wanted to publicly support in an HTTP library.